test/Hydration-test.js

```diff
@@ -103,9 +102,9 @@ describe('Hydration', function() {
   });
 
   it.skip('can calculate how many fluid ounces a user consumed on each day for the last 7 days', function() {
-    expect(userOneHydration.calculateOuncesConsumedOnDate('2019/06/17')).to.equal(96);
-    expect(userOneHydration.calculateOuncesConsumedOnDate('2019/06/16')).to.equal(69);
+    expect(userOneHydration.calculateWeeklyConsumption('2019/06/21')).to.deep.equal([37, 69, 96, 100, 20, 31, 91]);
```

Comment:
depending on how the display works, we may want to return an object here with date:ounces key value pairs instead of an array (just a slice of the hydrationCalendar property)